src/Fields/ArrayBlocks.jl

```diff
@@ -400,7 +400,7 @@ function  return_cache(k::TransposeMap,f::ArrayBlock{A,1} where A)
   ArrayBlock(g,collect(transpose(f.touched))),l
 end
 
-function  evaluate!(cache,k::TransposeMap,f::ArrayBlock)
+function  evaluate!(cache,k::TransposeMap,f::ArrayBlock{A,1} where A)
```

Comment:
This makes sense, it is a good thing that the signatures of `return_cache` and `evaluate!` are consistent one with the other.